src/skills/Attack.cpp

```diff
@@ -30,6 +30,7 @@ bt::Node::Status Attack::onUpdate() {
         return Status::Running;
     }
 
+
```

Comment:
I would rather remove this new line, because you already have a new line before it